Makefile

```diff
@@ -1,6 +1,7 @@
 
 NS ?= ghcr.io/playfab/
 
+export LISTENING_PORT=5000

```

Comment:
as mentioned in the comments, let's not tamper with the Makefile for now. Thanks!

Reply:
Removed listening_port field from makefile